lib/js/components/PopOver/PopOver.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-	<div>
-		<div v-if="triggerAction === POP_OVER_TRIGGER_ACTIONS.NONE">
+	<span>
+		<span v-if="triggerAction === POP_OVER_TRIGGER_ACTIONS.NONE">
```

Comment:
can we use `<template v-if>` to avoid unnecessary DOM nodes which can break styles?

Reply:
I think we can even use `<slot v-if`